docs/home.md

```diff
@@ -7,113 +7,67 @@
    :hidden:
 
    ovms_docs_quick_start_guide
-   ovms_docs_architecture
-   ovms_docs_models_repository
    ovms_docs_starting_server
-   ovms_docs_server_api
-   ovms_docs_clients
+   ovms_docs_server_app
    ovms_docs_dag
-   ovms_docs_binary_input
-   ovms_docs_model_cache
-   ovms_docs_metrics
-   ovms_sample_cpu_extension
-   ovms_docs_dynamic_input
    ovms_docs_stateful_models
-   ovms_docs_custom_loader
+   ovms_docs_binary_input
    ovms_docs_performance_tuning
-   ovms_docs_kubernetes
+   ovms_docs_advanced
    ovms_docs_demos
    ovms_docs_troubleshooting
 
 @endsphinxdirective
 
+OpenVINO&trade; Model Server (OVMS) is a high-performance system for serving models. Implemented in C++ for scalability and optimized for deployment on Intel architectures. OVMS uses the same architecture and API as [TensorFlow Serving](https://github.com/tensorflow/serving) and [KServe](https://github.com/kserve/kserve) while applying OpenVINO for inference execution. Inference service is provided via gRPC or REST API, making deploying new algorithms and AI experiments easy.
+
+![OVMS diagram](ovms_draft_diagram.png)
+
+## Model Serving
+
+A model server is a web server that hosts models and makes them accessible to software components over standart network protocols. Functionally it works similarly to a web server: you send a request and receive a response. Just like a web server, the model can be accessed across devices as long as they are connected via a common network.
 
 ![OVMS picture](ovms.png)
 
-OpenVINO&trade; Model Server (OVMS) is a high-performance system for serving machine learning models. It is based on C++ for high scalability 
-and optimized for Intel solutions, so that you can take advantage of all the power of the Intel® Xeon® processor or Intel’s AI accelerators 
-and expose it over a network interface. OVMS uses the same architecture and API as [TensorFlow Serving](https://github.com/tensorflow/serving), 
-while applying OpenVINO for inference execution. Inference service is provided via gRPC or REST API, making it easy to deploy new algorithms and AI experiments.
+The key advantage of a model server is its ability to process several client requests simultaneously so that the model uses less device memory. 
+As a result, you can create a lightweight client with only the necessary functions to perform API calls for edge or cloud deployments. 
```

Comment:
There is a note from JIRA ticker:
Most importantly we want to say why serving should be used, i.e. if you deploy in cluster or just want to disconnect app from inference, perhaps do model upgrades, etc.
It would be cool to get your comments on this info, should we include it and in which format.